public/downloads/index.html

```diff
@@ -72,198 +88,232 @@
           <div class="motto">
             <h1 class="title">Downloads</h1>
             <p style="font-size: 1.45em" class="description">
-             Find the resources you need to create with Cohort below.
+              Find the resources you need to create with Cohort below.
             </p>
             <br />
           </div>
         </div>
       </div>
     </div>
-    
-    
-       <div class="wrapper">
-        <div class="section text-center landing-section">
-          <div class="container">
-             <div class="row">
-
-              <div class="col-12">
-                <p class="custom-bigger-fontSize title">Web Client</p>
-              </div>
-              <div class="col-12">
-                  <p class="description">Web client description here</p>
-                </div>
 
-              <div class="col-xs-12 col-md-8 ml-auto mr-auto">
-                <a
-                  class="btn btn-block btn-outline-primary"
-                  role="button"
-                  alt="link to web client handbook"
-                  href="#">
-                    Link to web client handbook.
-                </a>
-              </div>
-              <div class="col-xs-12 col-md-8 ml-auto mr-auto mt-2">
-                  <a
-                    class="btn btn-block btn-outline-primary"
-                    role="button"
-                    alt="link to web client source files repo"
-                    href="#">
-                      Link to web client source files.
-                  </a>
-                </div>
-                <div class="col-xs-12 col-md-8 ml-auto mr-auto mt-2">
-                    <a
-                      class="btn btn-block btn-outline-primary"
-                      role="button"
-                      alt="link to demo event page"
-                      href="#">
-                        Link to demo event page.
-                    </a>
-                  </div>
+    <div class="wrapper">
```

Comment:
I don't think these work great as buttons, maybe because they're all quite different links/purposes and the heavy style makes them appear too similar... try stripping back a bit and let's see

Reply:
Ok, went with a more "link" styling.